nntrainer/models/neuralnet.cpp

```diff
@@ -1125,6 +1160,7 @@ int NeuralNetwork::train_run(
   auto epochs = getEpochs();
   ml_logd("[NNTrainer] Starts training. Current epoch: %d. Total epochs: %d.",
           epoch_idx + 1, getEpochs());
+  epoch_idx = 0;
```

Comment:
i wonder why initialize ```epoch_idx``` here ? 
Would it better to set or initialize ```epoch_idx``` before logging at (line1161) or through the constructor?

Reply:
Yes. It's my bad. It should be removed.